cypress/e2e/download.file.cy.ts

```diff
@@ -0,0 +1,20 @@
+/// <reference types ="Cypress" />
+
+import { DownloadPage } from "../pages";
+
+const downloadPage = new DownloadPage();
+
+describe("Download File", () => {
+  describe("Given the user is in teh Download Site ", () => {
+    beforeEach(() => {
+      downloadPage.goToDownloadSite();
+    });
+
+    describe("When the user downloads a file by clicking on the Download button", () => {
+      it("Then the file should be downloaded and it is correct", () => {
```

Comment:
```suggestion
      it("Then the file should be downloaded", () => {
```
you should validate the content as well